zboxcore/sdk/sdk.go

```diff
@@ -227,45 +230,28 @@ func GetReadPoolInfo(clientID string) (info *ReadPool, err error) {
 }
 
 // ReadPoolLock locks given number of tokes for given duration in read pool.
-func ReadPoolLock(tokens, fee int64, isOwner bool) (hash string, nonce int64, err error) {
+func ReadPoolLock(tokens, fee int64) (hash string, nonce int64, err error) {
 	if !sdkInitialized {
 		return "", 0, sdkNotInitialized
 	}
 
-	type lockRequest struct {
-		IsOwner    bool   `json:"is_owner"`
-		//MintTokens bool   `json:"mint_tokens"`
-	}
-
-	req := lockRequest{
-		IsOwner: isOwner,
-	}
-
 	var sn = transaction.SmartContractTxnData{
 		Name:      transaction.STORAGESC_READ_POOL_LOCK,
-		InputArgs: &req,
+		InputArgs: nil,
 	}
 	hash, _, nonce, err = smartContractTxnValueFee(sn, tokens, fee)
 	return
 }
 
 // ReadPoolUnlock unlocks tokens in expired read pool
-func ReadPoolUnlock(fee int64, isOwner bool) (hash string, nonce int64, err error) {
+func ReadPoolUnlock(fee int64) (hash string, nonce int64, err error) {
 	if !sdkInitialized {
 		return "", 0, sdkNotInitialized
 	}
 
-	type unlockRequest struct {
```

Comment:
will it work without request?

Reply:
yes, tested both manually and with custom system-test.